client/group.js

```diff
@@ -0,0 +1,18 @@
+Group = function(router, options) {
+  options = options || {};
+  this._router = router;
+  this.prefix = options.prefix || '';
+
+  this._middlewares = options.middlewares || [];
+};
```

Comment:
May be we need to introduce `group` prototype to here.
In that case, we could simply use this into the Router class. 
I'm back to my previous comment?

What do you think? I'm okay for both.